pkg/controllers/cloud_config_sync_controller_test.go

```
@@ -113,6 +135,20 @@ func makeManagedCloudConfig(platform configv1.PlatformType) *corev1.ConfigMap {
 	}, Data: map[string]string{"cloud.conf": defaultConfig}}
 }
 
+func makeManagedCloudConfigINI() *corev1.ConfigMap {
```

golangci-lint (2.11.4)

[error] func makeManagedCloudConfigINI is unused

(unused)